tools/Google.Cloud.Tools.ProjectGenerator/Program.cs

```diff
@@ -131,15 +131,15 @@ static int Main()
             {
                 ValidateCommonHiddenProductionDependencies();
                 var root = DirectoryLayout.DetermineRootDirectory();
-                var apis = ApiMetadata.LoadApis();
-                Console.WriteLine($"API catalog contains {apis.Count} entries");
+                var catalog = ApiCatalog.Load();
+                Console.WriteLine($"API catalog contains {catalog.Apis.Count} entries");
                 // Now we know we can parse the API catalog, let's reformat it.
-                ReformatApiCatalog();
-                RewriteReadme(apis);
-                RewriteDocsRootIndex(apis);
-                HashSet<string> apiNames = new HashSet<string>(apis.Select(api => api.Id));
+                ReformatApiCatalog(catalog);
+                RewriteReadme(catalog.Apis);
```

Comment:
nit: consider passing the catalog here and in the next one, even if you can do with the list here.